OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -27,6 +27,10 @@ class BaseBuilderQueueManager
 		readonly PowerManager playerPower;
 		readonly PlayerResources playerResources;
 
+		// need this to avoid crash in WaterCheck when there is no NavalProductionTypes
+		// TODO: remove it when related crash is fixed.
```

Comment:
I honestly think this should actually stay even if we find and fix the issue in `IsAreaAvailable`, so in my opinion just drop the TODO.

Reply:
ok, if you say so.